### ISSUE TITLE
How to «Blizzard.js»

### ISSUE THREAD

**Issue by AlexZeDim** [benweier/blizzard.js#84]
Hi Ben!

Let's be honest, according to **`readme.md`** I should just copy/paste Step 1 & Step 2 and then receive an pretty accurate json with character data. But instead of this, anyone who will be doing the same steps receive only 401 error in console.log.

So, all those, who already checked the code or use this module for quite a long time already knows that there should be 'token' param, just like this:

```
const blizzard = require('blizzard.js').initialize({
    key: 'key',
    secret: 'secret',
});

blizzard.getApplicationToken()
    .then(response => {
        blizzard.defaults.token = response.data.access_token
        blizzard.wow.character(['profile'], { origin: 'us', realm: 'amanthul', name: 'charni' })
            .then(response => {
                console.log(response.data);
            });
    });
```

But I'm not sure that I am doing it 'right'. Probably there is a better way to import my 'token' during **`.initialize`** phase. So maybe we should update/refresh the docs?

**Comment by Ziao** [comment]
Cheers! Didn't dive into the source yet, but thanks to you I won't have to :)

**Comment by Rinoa1337** [comment]
Thx man!

**Comment by AlexZeDim** [comment]
This issue is also fixed with [PR#95](https://github.com/benweier/blizzard.js/pull/95)

**Comment by hbence01** [comment]
Thanks, I was actually wondering why I got 401 👍
